tests/post_training/pipelines/base.py

```diff
@@ -92,9 +92,10 @@ def fill(self, stdout: str) -> None:
 
 @dataclass
 class NumCompressNodes:
-    num_fq_nodes: Optional[int] = None
-    num_int8: Optional[int] = None
-    num_int4: Optional[int] = None
+    num_fq_nodes: int = 0
+    num_int8: int = 0
+    num_int4: int = 0
+    num_sparse_activations: int = 0
```

Comment:
Does it have to be defined in the `base.py`? I believe a test pipeline should define it's own fields here.

Reply:
I think so, it makes code common. If any test does not need any field, that's fine, only values in reference data matter